src/components/con/common/Button.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classnames from 'classnames';
+import { Link } from 'gatsby';
+
+export interface ButtonProps extends React.ButtonHTMLAttributes<HTMLDivElement> {
+  size?: 'small' | 'large';
+  className?: string;
+  empty?: boolean;
+  disabled?: boolean;
+  to?: string;
+}
+
+const Button: React.ComponentType<ButtonProps> = ({
```

Comment:
React.FC type is better i guess (ComponentType can be either a class component or a functionnal component)